### PR TITLE
Review PnP Service Samples

### PR DESCRIPTION
**Pull request by rido-min**
## Purpose
Review the service samples for PnP, to make sure they are aligned with docs.
https://github.com/MicrosoftDocs/azure-docs-pr/pull/132218/files
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check

* PnP samples for components work on both sides (device and service)

### CODE REVIEW

File: iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs [hunk 1]
```diff
@@ -114,20 +115,16 @@ private async Task InvokeRebootCommandAsync()
 
         private async Task UpdateDigitalTwinComponentPropertyAsync()
         {
+
```
Comment (drwill-ms) [inline iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs:118]: remove empty newline

File: iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs [hunk 2]
```diff
@@ -114,20 +115,16 @@ private async Task InvokeRebootCommandAsync()
...
             int desiredTargetTemperature = Random.Next(0, 100);
 
-            const string targetTemperaturePropertyName = "targetTemperature";
-            string propertyUpdate = CreatePropertyPatch(targetTemperaturePropertyName,
-                JsonConvert.SerializeObject(desiredTargetTemperature),
-                Thermostat1Component);
-            var twinPatch = new Twin();
-            twinPatch.Properties.Desired[targetTemperaturePropertyName] = desiredTargetTemperature;
+            var twinPatch = CreatePropertyPatch(targetTemperaturePropertyName, desiredTargetTemperature, Thermostat1Component);
+            _logger.LogDebug($"Update the {targetTemperaturePropertyName} property under component {Thermostat1Component} on the {_digitalTwinId} digital twin to { desiredTargetTemperature}.");
```
Comment (drwill-ms) [inline iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs:125]: Verb tense should be "Updating" or "Updated" depending on what tense we're following in the sample.

File: iot-hub/Samples/service/RegistryManagerSample/RegistryManagerSample.cs [hunk 3]
```diff
@@ -26,6 +26,7 @@ public RegistryManagerSample(RegistryManager registryManager)
             _registryManager = registryManager ?? throw new ArgumentNullException(nameof(registryManager));
         }
 
+
```
Comment (drwill-ms) [inline iot-hub/Samples/service/RegistryManagerSample/RegistryManagerSample.cs:29]: remove extra newline

**Review by drwill-ms — APPROVED** [review]
Minor style changes, and otherwise okay

File: iot-hub/Samples/service/DeviceStreamingSample/ServiceClientStreamingSample.csproj [hunk 4]
```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-  	<PackageReference Include="Microsoft.Azure.Devices" Version="1.27.0-preview-004" />
+  	<PackageReference Include="Microsoft.Azure.Devices" Version="1.27.2" />
```
Comment (barustum) [inline iot-hub/Samples/service/DeviceStreamingSample/ServiceClientStreamingSample.csproj:18]: Device Streaming Sample is a preview only feature. Updating the NuGet package to a GA package might break the sample.

**Pull request merged**